File: C# Part 1/Operators And Expressions/PrimeCheck/PrimeCheck.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace PrimeCheck
{
    class PrimeCheck
    {
        static void Main()
        {
            int number = int.Parse(Console.ReadLine());
            bool prime = true;
            if (number > 0 && number <= 100)
            {
                for (int divider = 2; divider <= Math.Sqrt(number); divider++)
                {
                    if (number % divider == 0)
                    {
                        prime = false;
                        break;
                    }
                }
                Console.WriteLine(prime ? "true" : "false");
            }
            else
            {
                Console.WriteLine("false");
            }

        }
    }
}

File: JavaScript Fundamentals/Operators and expressions/PrimeCheck.js
function solve(args) {
    var number = +args[0];

    if (number < 2) {
        console.log('false');
    }
    else {
        var prime = true;
        for (var divider = 2; divider <= Math.sqrt(number); divider += 1) {
            if (number % divider === 0) {
                prime = false;
                break;
            }
        }
         console.log(prime);
    }
}